src/sentry/models/__init__.py

```diff
@@ -1,3 +1,4 @@
+from ..users.models.authenticator import *  # NOQA
```

Comment:
```suggestion
from sentry.users.models.authenticator import *  # NOQA
```

Lets stick to absolute imports. They are easier to understand and more portable.